src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -66,4 +66,8 @@ public CommandResult execute() {
         }
     }
 
+    @Override
```

Comment:
Update your header comment, since overriden method has a different behavior from the parent class.